### PR TITLE
Add Markdown support for entry comment email

### PR DESCRIPTION
**Pull request by thenav56**

### CODE REVIEW

**Review by AdityaKhatri — APPROVED** [review]

File: apps/analysis_framework/serializers.py [hunk 1]
```diff
@@ -182,6 +187,24 @@ class Meta:
...
         fields = ('__all__')
 
+    def get_projects(self, obj):
+        user = None
+        if 'request' in self.context:
+            user = self.context['request'].user
+        projects = obj.project_set.exclude(Q(is_private=True) & ~Q(members=user))
+        return SimpleProjectSerializer(projects, many=True, read_only=True).data
+
+    def get_can_add_users(self, obj):
```
Comment (bewakes) [inline apps/analysis_framework/serializers.py]: better to have a different name. The name sounds like it has boolean value.

File: apps/analysis_framework/serializers.py [hunk 2]
```diff
@@ -170,6 +172,9 @@ class AnalysisFrameworkSerializer(RemoveNullFieldsMixin,
     )
 
     is_admin = serializers.SerializerMethodField()
+    can_add_users = serializers.SerializerMethodField()
+    projects = serializers.SerializerMethodField()
```
Comment (bewakes) [inline apps/analysis_framework/serializers.py]: The names `projects` and `projects_count` refer to different sets of projects. Good to have a more descriptive name, something like private projects or all_projects_count.

**Review by bewakes — CHANGES_REQUESTED** [review]

File: apps/analysis_framework/serializers.py (hunk 1 above)
Comment (thenav56) [reply apps/analysis_framework/serializers.py]: @bewakes Let me know if you have suggestions.

`users_with_can_add_permission` sounds good?

cc: @AdityaKhatri

File: apps/analysis_framework/serializers.py (hunk 2 above)
Comment (thenav56) [reply apps/analysis_framework/serializers.py]: @bewakes Does `all_projects_count` and `visible_projects` sounds good?

cc: @AdityaKhatri

Comment (bewakes) [reply apps/analysis_framework/serializers.py]: Yes, sounds perfect!!

File: apps/analysis_framework/serializers.py (hunk 1 above)
Comment (bewakes) [reply apps/analysis_framework/serializers.py]: Maybe `users_with_add_permission`?

Comment (thenav56) [reply apps/analysis_framework/serializers.py]: Sounds good.

Comment (thenav56) [reply apps/analysis_framework/serializers.py]: Changes pushed https://github.com/the-deep/server/pull/470/commits/643fe281782d0cec95d2c7b114ed1734bd22be78

File: apps/analysis_framework/serializers.py (hunk 2 above)
Comment (thenav56) [reply apps/analysis_framework/serializers.py]: Changes pushed https://github.com/the-deep/server/pull/470/commits/643fe281782d0cec95d2c7b114ed1734bd22be78

**Review by bewakes — APPROVED** [review]

**Pull request merged**